draft-eckert-ietf-and-energy-overview.mkd

```diff
@@ -584,29 +660,40 @@ specifications, information models, cross-adaptation layer specification (such a
 specifications) and maintenance and informational documents for other pre-existing IETF
 work in this space. 
 
+Notably, a key specification produced is {{RFC7668}}, "IPv6 over BLUETOOTH(R) Low Energy", using IPv6 over Low-power Wireless Personal Area Network (6LoWPAN) techniques, as well as the related {{RFC9159}} for the formation of extended topologies, an IPv6 mesh over Bluetooth LE links.
+
+From a management perspective, produced {{RFC7388}}, LOWPAN MIB, as well as several specific improvements around power such as {{RFC7973}}, {{RFC8025}}, {{RFC8928}}, {{RFC8931}}, and {{RFC9034}}.
+
+Finally, the 6LO WG also produced {{RFC8105}}, IPv6 over DECT Ultra-Low Energy, and {{RFC9354}}, IPv6 over Power Line Communication (PLC).
+
+
 ### ROLL WG
 
 In the RouTinG (RTG) area of the IETF, the "Routing Over Low power and Lossy networks" (ROLL) WG
 has produced since 2008 23 RFC. Initially it produced requirement RFCs of different type of
 "Low-power and Lossy Networks": urban: {{RFC5548}}, industrial {{RFC5673}}, home automation
 {{RFC5826}} and building automation {{RFC5867}}.
 
- Since then its work is mostly focused
-on the "IPv6 Routing Protocol for Low-Power and Lossy Networks" (RPL) {{RFC6550}}, which
-is used in a wide variety of the above described IPv6 instances of LLN networks
+Since then, its work is mostly focused
+on the "IPv6 Routing Protocol for Low-Power and Lossy Networks" (RPL) {{RFC6550}} {{RFC6551}} {{RFC6552}}, which
```

Comment:
```suggestion
on the "IPv6 Routing Protocol for Low-Power and Lossy Networks" (RPL) {{RFC6550}} {{RFC6551}}{{RFC6552}}, which
```

Reply:
I'd leave the extra space between references.